setup/src/common/scripts/onCreateCommand.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+# Copyright (c) 2022-2024 Contributors to the Eclipse Foundation
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+sudo chmod +x .devcontainer/scripts/*.sh
+sudo chown -R $(whoami) $HOME
+
+.devcontainer/scripts/setup-git.sh
+
+if [[ -z "${VELOCITAS_OFFLINE}" ]]; then
+    .devcontainer/scripts/configure-codespaces.sh
+    .devcontainer/scripts/upgrade-cli.sh
+fi
+
+echo "#######################################################"
+echo "### Run VADF Lifecycle Management                   ###"
+echo "#######################################################"
+velocitas init
```

Comment:
In the `onCreate` of the Python template there was this workaround:
```
# needed to get rid of old leftovers
sudo rm -rf ~/.velocitas
```
Isn't that needed anymore?

Reply:
Its not required anymore, Andre checked it some weeks back.